StarTable format specification.md

```diff
@@ -142,8 +142,8 @@ functionality. The different block types are summarized in this table:
 
 | Block type    | Start marker first-column cell content                       | End marker                                             | Description & remarks                                        |
 | ------------- | ------------------------------------------------------------ | ------------------------------------------------------ | ------------------------------------------------------------ |
-| Directive     | `***` followed by directive block name                       | - Empty first column cell; or<br>- New block start     | Placeholder for a variety of functions e.g.  <br> - Version control <br> - Allowing tables from various sources to be added dynamically to the set of tables statically present in a sheet |
-| Table         | `**`  followed by table name                                               (which may not start with `*`) | - Empty first column cell; or <br> - New block start   | Primary data content                                         |
+| Directive     | `***` followed by directive block name (which may not contain `*`) | - Empty first column cell; or<br>- New block start     | Placeholder for a variety of functions e.g.  <br> - Version control <br> - Allowing tables from various sources to be added dynamically to the set of tables statically present in a sheet |
+| Table         | `**`  followed by table name                                               (which may not contain `*`) | - Empty first column cell; or <br> - New block start   | Primary data content                                         |
```

Comment:
No need for the "which may not contain`*`" -- this is already explicit in the "symbol string"-specification.